src/projections/auto-data-key.js

```diff
@@ -0,0 +1,18 @@
+import { capitalizeFirstLetter } from '../utils';
```

Comment:
as we discussed, we should use `key` instead of `data-key`. Shall we change the file name to `auto-key`?